pyopae/pyproperties.cpp

```diff
@@ -63,31 +63,33 @@ const char *properties_doc_get() {
 
       type (fpga_objtype): The object type - DEVICE or ACCELERATOR.
 
-      bus : The PCIe bus numer.
+      bus (uint8_t) : The PCIe bus numer.
 
-      device : The PCIe device number.
+      device (uint8_t) : The PCIe device number.
 
-      function : The PCIe function number.
+      function (uint8_t) : The PCIe function number.
 
-      socket_id: The socket ID encoded in the FIM.
+      socket_id (uint8_t): The socket ID encoded in the FIM.
 
-      num_slots: Number of slots available in the FPGA.
+      num_slots (uint32_t): Number of slots available in the FPGA.
 
-      num_errors: Number of error registers in the resource.
+      num_errors (uint32_t): Number of error registers in the resource.
 
       bbs_id (uint64_t): The BBS ID encoded in the FIM.
 
-      bbs_version: The version of the BBS.
+      bbs_version (tuple): The version of the BBS.
 
-      vendor_id: The vendor ID in PCI config space.
+      vendor_id (uint16_t): The vendor ID in PCI config space.
+
+      device_id (uint16_t): The device ID in PCI config space.
 
       model (str): The model of the FPGA.
 
-      local_memory_size: The size (in bytes) of the FPGA local memory.
+      local_memory_size (uint64_t): The size (in bytes) of the FPGA local memory.
```

Comment:
Do we support this property currently?  Last I remembered, it was a TODO item.

Reply:
It looks like it's still not supported 😞 
In this case, the C++ backing code will throw a `not_supported` exception which will get converted into a Python RuntimeError with the message generated by the `not_supported` exception.
I think this is fine for now but we should probably look at either defining analogous exception classes in Python or convert the C++ exceptions to standard Python exceptions that make sense for the situation.